.github/ISSUE_TEMPLATE/release-template.md

```diff
@@ -0,0 +1,49 @@
+---
+name: Release Process
+about: COMMITTER ONLY: Managing the Jetty release process
+title: 'Jetty Releases 9.4.x, 10.0.y, 11.0.y'
+assignees: ''
+labels: Build
+
+---
+
+**Jetty Versions:**
+This release process will produce releases:
+
+**Target Date:**
+
+**Tasks:**
+- [x] Create the release(s) issue.
+- [ ] Update the target Jetty version(s) in the issue.  
+- [ ] Update the target release date in the issue.
+- [ ] Link this issue to the target [project(s)](https://github.com/eclipse/jetty.project/projects).
+- [ ] Assign this issue to a "release manager".
+- [ ] Review [draft security advisories](https://github.com/eclipse/jetty.project/security/advisories). Ensure that issues are created and assigned to GitHub Projects to capture any advisories that will be announced.
+- [ ] Review their issues/PRs assigned to the target [project(s)](https://github.com/eclipse/jetty.project/projects).  Any PRs that are moved to subsequent releases should be commented on so their authors are informed.
+- [ ] Freeze the target [project(s)](https://github.com/eclipse/jetty.project/projects) by editing their names to "Jetty X.Y.Z FROZEN"
+- [ ] Wait 24 hours from last change to the issues/PRs included in a FROZEN GitHub Project.
+- [ ] Verify target [project(s)](https://github.com/eclipse/jetty.project/projects) are complete.
+- [ ] Verify that branch `jetty-10.0.x` is merged to branch `jetty-11.0.x`.
+- [ ] Assign issue to "build manager", who will stage the releases.
+   + [ ] Ensure `VERSION.txt` additions for each release will be meaningful, descriptive, correct text.
+   + [ ] Stage Release(s).
```

Comment:
This misses the bit that Jetty 9.4.x should be built with Java 11. Why that must be so, I have no idea. @joakime?